lib/protobuf/field/bool_field.rb

```diff
@@ -17,7 +17,15 @@ def self.default
       # #
 
       def acceptable?(val)
-        [true, false].include?(val)
+        [true, false].include?(val) || (val.is_a?(String) && %w(true false).include?(val))
+      end
+
+      def coerce!(val)
+        if val.is_a?(String)
```

Comment:
I tend to dislike checking for particular classes since that defeats the purpose of duck typing. Could do something as simple as `val.to_s == 'true'` if you wanted?